### ISSUE TITLE
All links to css files should have "deployment" parameter.

### ISSUE THREAD

**Issue by noisy** [noisy/steemprojects.com#31]
To prevent the problems with caching css files by browsers, after each deployment, parameter should be added to all css files, as follows:

```
<link rel="stylesheet" href="{{ STATIC_URL }}css/profile.css?deployment={{ DEPLOYMENT_DATATIME }}" type="text/css" />
```

**Comment by andrejcibik** [comment]
I use .css?v=n+1 
Its shorter :)

**Comment by noisy** [comment]
this is already implemented, just noticed, that no styles use this technique.

fixed with 2533c1d355479d0ee95fe3640febb3313dd7b40c
